src/main/Excavator.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+public class Excavator extends Machine {
+ @Override
+ public void doWork() {
+    System.out.println("Excavator started working")
+ }
+ @Override
+ public void stopWork() {
+    System.out.println("Excavator stopped working")
```

Comment:
There is a missing semicolon at the end of this line. Java statements must end with a semicolon.